src/components/custom-aggrid/custom-tooltip-values.tsx

```diff
@@ -0,0 +1,38 @@
+import { Grid, Typography, Tooltip } from '@mui/material';
+import { ITooltipParams } from 'ag-grid-community';
+import { FunctionComponent } from 'react';
+
+const CustomTooltipValues: FunctionComponent<ITooltipParams> = (props) => {
+    const values: string[] = props?.value?.values;
+    return (
+        <>
+            {values && values.length && (
```

Comment:
I think it'd better to use values?.length

Reply:
Done